src/app/backend/apihandler.go

```diff
@@ -167,7 +167,8 @@ func CreateHttpApiHandler(client *client.Client) http.Handler {
 }
 
 type ApiHandler struct {
-	client *client.Client
+	client         *client.Client
+	heapsterClient *client.Client
```

Comment:
HeapsterClient should be RESTClient, not client.client. The latter is for services, pods, etc.